docker-compose.yml

```diff
@@ -1,23 +1,23 @@
 version: '2.3'
 
 x-lagoon-project:
-  &lagoon-project govcms-saas
+  &lagoon-project govcms7-scaffold
 
 x-lagoon-local-dev-url:
-  &lagoon-local-dev-url http://govcms-saas.docker.amazee.io
+  &lagoon-local-dev-url http://govcms7-scaffold.docker.amazee.io
 
 x-volumes:
   &default-volumes
     volumes:
-      - ./themes:/app/profiles/govcms/themes/custom:${VOLUME_FLAGS:-delegated}
+      - ./themes:/app/sites/default/themes/custom:${VOLUME_FLAGS:-delegated}
```

Comment:
This shouldn't be a problem but requires a govcmslagoon update to alter the redirect helper: https://github.com/govCMS/govcmslagoon/blob/master/.docker/images/nginx/helpers/acsf_redirects.conf#L4

Reply:
Resolved with https://github.com/govCMS/govcmslagoon/pull/43